nomad/local/grapl-local-infra.nomad

```diff
@@ -71,13 +71,8 @@ job "grapl-local-infra" {
       driver = "docker"
 
       config {
-        image        = "redis:latest"
-        ports        = ["redis"]
-        network_mode = "grapl-network"
```

Comment:
We're no longer using the docker grapl-network or the associated network aliases